test/e2e/all_test.go

```diff
@@ -81,6 +81,7 @@ func TestAll(t *testing.T) {
 		t.Run("TestRouteMetricsControllerRouteAndNamespaceSelector", TestRouteMetricsControllerRouteAndNamespaceSelector)
 		t.Run("TestSetIngressControllerResponseHeaders", TestSetIngressControllerResponseHeaders)
 		t.Run("TestSetRouteResponseHeaders", TestSetRouteResponseHeaders)
+		t.Run("TestDashboardCreation", TestDashboardCreation)
```

Comment:
As `TestDashboardCreation` mutates the global `infracConfig` object we should move this test to the "serial" suite.